tests/util_test.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from core.util import postcode_to_itl1
+
+
+def test_postcode_to_itl1_returns_itl1():
+    itl1 = postcode_to_itl1("DT1 8PD")
+    assert itl1 == "TLK"
+
+
+def test_postcode_to_itl1_returns_itl1_lowercase():
+    itl1 = postcode_to_itl1("dt1 8pd")
+    assert itl1 == "TLK"
+
+
+def test_postcode_to_itl1_returns_itl1_leading_space():
+    itl1 = postcode_to_itl1(" dt1 8pd")
+    assert itl1 == "TLK"
+
+
+def test_postcode_to_itl1_returns_itl1_trailing_space():
+    itl1 = postcode_to_itl1("dt1 8pd ")
+    assert itl1 == "TLK"
+
+
```

Comment:
Can we test with a single letter example e.g. `N1 0AA` should return `TLI`?

Reply:
yeah nice spot